npc/re/quests/illusion_investigation.txt

```diff
@@ -306,92 +306,92 @@ pay_d03_i,149,34,5	script	Jiroker#ziroker	4_M_ORIENT02,{
 OnInit:
 	questinfo( QTYPE_QUEST, QMARK_YELLOW, "BaseLevel >= 100 && isbegin_quest(3470) == 2 && isbegin_quest(3446) == 0" );
 
-	questinfo( QTYPE_EVENT2, QMARK_YELLOW, "checkquest(3447,HUNTING) == 2" );
+	questinfo( QTYPE_QUEST2, QMARK_YELLOW, "checkquest(3447,HUNTING) == 2" );
 	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "isbegin_quest(3446) == 2 && checkquest(3448,PLAYTIME) == -1 && checkquest(3447,HUNTING) == -1" );
 	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "checkquest(3448,PLAYTIME) == 2" );
 	end;
 }
 
-ice_dun02,142,18,5	script	Allysia#elisia	4_F_OPERATION,{
+ice_dun02,142,18,5	script	Elisia#elisia	4_F_OPERATION,{
```

Comment:
went back to the name Elisia?

Reply:
^`After discussion, https://github.com/rathena/rathena/commit/9ddd26f1e2dc408b3a667c306df401fd04e04384 reverted to not using same NPC name as Kiel quest.`

Reply:
This one should be "Elysia". It's acommon name